Ginger/Ginger/ApplicationModelsLib/POMModels/POMEditPage.xaml.cs

```diff
@@ -116,6 +117,14 @@ public POMEditPage(ApplicationPOMModel POM, RepositoryItemPageViewMode editMode
 
             mScreenShotViewPage = new ScreenShotViewPage(mPOM.Name, source);
             xScreenShotFrame.Content = mScreenShotViewPage;
+            if (mPOM.PageLoadFlow == ApplicationPOMModel.ePageLoadFlow.PageURL)
+            {
+                xRDBPageURL.IsChecked = true; 
+            }
```

Comment:
if you set binding above so why you need also to update the selected radio button here?
anyway i don't like mPOM.IsPageLoadBusinessFlow because in future we might have more page load flow types.
instead just set the selected radio button on load based on mPOM.PageLoadFlow enum value and then catch radio button selectionChanged event and update the relevent enum value on mPOM.PageLoadFlow

Reply:
done